palantir-java-format/src/test/resources/com/palantir/javaformat/java/testdata/ExpressionSwitch.output

```diff
@@ -47,4 +47,22 @@ class ExpressionSwitch {
             default -> false;
         };
     }
+
+    public void test1(int y) {
+        int x =
+                switch (y) {
+                    case 1 -> 1;
+                    case 2 -> throw new IllegalArgumentException();
+                    default -> throw new IllegalStateException();
+                };
```

Comment:
Would like to fix this in the future FWIW to instead be:
```java
        int x = switch (y) {
                case 1 -> 1;
                case 2 -> throw new IllegalArgumentException();
                default -> throw new IllegalStateException();
        };
```
But that requires more complicated changes to happen to the `visitAssignment` method which should be left to a separate PR